frontend/components/commons/header/filters/FiltersList.vue

```diff
@@ -235,6 +235,22 @@ export default {
             a.key.toLowerCase() > b.key.toLowerCase() ? 1 : -1
           )) ||
         [];
+      const dateFields = [
+        {
+          id: "last_updated",
+          key: "last_updated",
+          group: "Sort",
+          name: "Last updated",
+          disabled: this.recordPropertyHasValue("last_updated"),
+        },
+        {
+          id: "event_timestamp",
+          key: "event_timestamp",
+          group: "Sort",
+          name: "Event timestamp",
+          disabled: this.recordPropertyHasValue("event_timestamp"),
+        },
+      ].filter(({ disabled }) => !disabled);
```

Comment:
code cleanning : there is a duplicated code to create the array => use a method with a property to difene wich object to return (with last_update or event_timestamp